examples/mysql-kopia-repository-server/README.md

```diff
@@ -0,0 +1,423 @@
+
+kubectl create secret generic repo-pass -n kanister --from-literal=repo-password=<repository_password_set_while_creating_kopia_repository>
+
+kubectl create -f s3_location_creds.yaml -n kanister
+
+kubectl create -f s3_location.yaml -n kanister
+```
+
+- Apply Repository Server CRD
+```bash
+$ kubectl create -f pkg/customresource/repositoryserver.yaml -n kanister
+```
+
+- Create Repository Server CR
+```bash
+vi repo-server-cr.yaml 
+```
+```
+apiVersion: cr.kanister.io/v1alpha1
+kind: RepositoryServer
+metadata:
+  labels:
+    app.kubernetes.io/name: repositoryserver
+    app.kubernetes.io/instance: repositoryserver-sample
+    app.kubernetes.io/part-of: kanister
+    app.kuberentes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: kanister
+  name: kopia-repo-server-1
+  namespace: kanister
+spec:
+  storage:
+    secretRef:
+      name: s3-location
+      namespace: kanister
+    credentialSecretRef:
+      name: s3-creds
+      namespace: kanister
+  repository:
+    rootPath: /repo-controller/
+    passwordSecretRef:
+      name: repo-pass
+      namespace: kanister
+    username: kanisterAdmin
+    hostname: mysql.app
+  server:
+    adminSecretRef:
+      name: repository-admin-user
+      namespace: kanister
+    tlsSecretRef:
+      name: repository-server-tls-cert
+      namespace: kanister
+    userAccess:
+      userAccessSecretRef:
+        name: repository-server-user-access
+        namespace: kanister
+      username: kanisteruser
+```
+```bash
+$ kubectl create -f repo-server-cr.yaml -n kanister
+```
+**NOTE:**
+
+Make Sure the Repository Server is in ServerReady State before creating actionsets.
+You could check the status of the Repository Server CR by running following command
+```bash
+$ kubectl get repositoryservers.cr.kanister.io kopia-repo-server-1 -n kanister -o yaml
+```
+
+**NOTE:**
+
+The above command will configure a kopia repository server, which manages artifacts resulting from Kanister
+data operations such as backup should go. This is stored as a `repositoryservers.cr.kanister.io`
+*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
+requires a Repository Server reference to complete the action. This CR (`repositoryservers.cr.kanister.io`)
+can be shared between Kanister-enabled application instances.
+
+### Create Blueprint
+
+Create Blueprint in the same namespace as the Kanister controller
+
+```bash
+$ kubectl create -f ./mysql-blueprint.yaml -n kanister
+```
+
+Once MySQL is running, you can populate it with some data. Let's add a table called "pets" to a test database:
+
+```bash
+# Connect to MySQL by running a shell inside MySQL's pod
+$ kubectl exec -ti $(kubectl get pods -n mysql --selector=app.kubernetes.io/instance=mysql-release -o=jsonpath='{.items[0].metadata.name}') -n mysql -- bash
+
+# From inside the shell, use the mysql CLI to insert some data into the test database
+# Create "test" db
```

Comment:
Is this a command to create the database? If yes, it's incorrect I think.
Should this not be after we have logged into the MySQL instance using mysql  command?

Reply:
This is just headline, the commands are below that